config/locales/en.yml

```diff
@@ -334,7 +335,7 @@ en:
     comment_text: "<p><i>%{comment}</i></p>"
     comment_link: "<p>To read it, %{link}!</p>"
     new_group_subject: "if me | %{user} created a group \"%{group}\""
-    new_group_body: "<p>%{user} created a group \"%{group}\":</p><p><i>%{description}</i></p><p>To learn more and join, %{link}!</p>"
+    new_group_body: "<p>%{user} created a group \"%{group}\":</p><p><i>%{description}</i></p><p>To learn more and join, %{link}!</p><p>To see our code of conduct %{link_to_code_of_conduct}!</p>"
```

Comment:
We should change the copy from`To see our code of conduct %{link_to_code_of_conduct}!` to `Please follow our %{link_to_code_of_conduct}!` where link_to_code_of_conduct is `link_to(t('navigation.code_of_conduct'), 'https://www.contributor-covenant.org/')`